examples/image_streamer/artifact_bundle.rb

```diff
@@ -20,7 +20,9 @@
 
 i3s_client = {
   url: ENV['I3S_URL'],
-  oneview_client: oneview_client
+  oneview_client: oneview_client,
+  api_version: 1600,
+  ssl_enabled: false
```

Comment:
I am not sure if ssl_enabled should be false by default, can you check for other examples, and follow a similar patter please?

Reply:
done